airsonic-main/src/main/java/org/airsonic/player/service/PlaylistService.java

```diff
@@ -27,13 +27,12 @@
 import org.airsonic.player.domain.MediaFile;
 import org.airsonic.player.domain.PlayQueue;
 import org.airsonic.player.domain.Playlist;
-import org.airsonic.player.domain.User;
 import org.airsonic.player.service.playlist.PlaylistExportHandler;
 import org.airsonic.player.service.playlist.PlaylistImportHandler;
 import org.airsonic.player.util.FileUtil;
 import org.airsonic.player.util.Pair;
 import org.airsonic.player.util.StringUtil;
-import org.apache.commons.io.FilenameUtils;
+//import org.apache.commons.io.FilenameUtils;
```

Comment:
This can be deleted I think

Reply:
It could be, but I left it in but commented out so its there whenever a solution to the playlist import issue is decided.